addons/l10n_pe_website_sale/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    "name": "Peruvian eCommerce",
+    "countries": ["pe"],
+    "version": "0.1",
+    "summary": "Be able to see Identification Type in ecommerce checkout form.",
+    "category": "Accounting/Localizations/Website",
+    "author": "Vauxoo, Odoo",
+    "license": "LGPL-3",
+    "depends": [
+        "website_sale",
+        "l10n_pe",
+        "payment_custom",
```

Comment:
Why do we depend on payment_custom?

Reply:
I needed this module installed first to be able to publish it and change its redirect_form_view_id, as you can see [here](https://github.com/odoo/odoo/blob/5ac2008a0535e76b04ee2fa16cea0d641a6b99e8/addons/l10n_pe_website_sale/demo/website_demo.xml#L13). Otherwise changing those fields would result in an error and Julien asked to set up a payment method.

Reply:
Oh, I missed this part. It's probably better to not add an extra dependency just for the sake of a demo data. What do you think @jco-odoo ? Anyway we can set up the payment method manually, it's not too annoying